ansible/config_sonic_basedon_testbed.yml

```diff
@@ -730,6 +730,20 @@
         become: true
         shell: config bgp startup all
 
+      - name: Copy smartswitch dpu config
+        copy: src=golden_config_db/smartswitch_dpu_extra.json
+              dest=/tmp/dpu_extra.json
+        become: true
+        when: topo in ["t1-smartswitch-ha","t1-28-lag","smartswitch-t1"]
```

Comment:
Did you verify all the topo?

Reply:
verified smartswitch-t1. Let me verify others and get back